File: samples/client/petstore/java/retrofit2/src/main/java/org/openapitools/client/api/DefaultApi.java
package org.openapitools.client.api;

import org.openapitools.client.CollectionFormats.*;

import retrofit2.Call;
import retrofit2.http.*;

import okhttp3.RequestBody;
import okhttp3.ResponseBody;
import okhttp3.MultipartBody;

import java.math.BigDecimal;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Set;

public interface DefaultApi {
  /**
   * 
   * get some object
   * @return Call&lt;List&lt;BigDecimal&gt;&gt;
   */
  @GET("example/someMethod")
  Call<List<BigDecimal>> retrieveSomething();
    

}

File: samples/client/petstore/kotlin-retrofit2/src/main/kotlin/org/openapitools/client/apis/DefaultApi.kt
package org.openapitools.client.apis

import org.openapitools.client.infrastructure.CollectionFormats.*
import retrofit2.http.*
import retrofit2.Call
import okhttp3.RequestBody
import com.squareup.moshi.Json


interface DefaultApi {
    /**
     * 
     * get some object
     * Responses:
     *  - 200: The response with results
     *
     * @return [Call]<[kotlin.Int]>
     */
    @GET("example/someMethod")
    fun retrieveSomething(): Call<kotlin.Int>

}
